arangod/ClusterEngine/ClusterRestWalHandler.cpp

```diff
@@ -105,42 +106,33 @@ void ClusterRestWalHandler::flush() {
   }
 
   bool waitForSync = false;
-  bool waitForCollector = false;
+  bool flushColumnFamilies = false;
 
   if (slice.isObject()) {
     // got a request body
-    VPackSlice value = slice.get("waitForSync");
+    VPackSlice value = slice.get(StaticStrings::WaitForSyncString);
     if (value.isString()) {
-      waitForSync = (value.copyString() == "true");
+      waitForSync = (value.stringView() == "true");
     } else if (value.isBoolean()) {
       waitForSync = value.getBoolean();
     }
 
     value = slice.get("waitForCollector");
```

Comment:
Also define a StaticString for "waitForCollector"?

Reply:
I am a bit reluctant to define a static string value for an attribute that has the wrong name anyway.

Reply:
I see pros and cons for both - so feel free to ignore this comment then.